@xen-orchestra/backups/RemoteAdapter.js

```diff
@@ -576,20 +576,29 @@ class RemoteAdapter {
     return stream
   }
 
-  async readDeltaVmBackup(metadata) {
+  async readDeltaVmBackup(metadata, ignoredVdis) {
     const handler = this._handler
     const { vbds, vdis, vhds, vifs, vm } = metadata
     const dir = dirname(metadata._filename)
 
+    const filteredVdis =
+      ignoredVdis === undefined
+        ? vdis
+        : Object.entries(vdis).reduce(
+            (filteredVdis, [vdiRef, vdi]) =>
+              ignoredVdis.has(vdi.uuid) ? filteredVdis : { ...filteredVdis, [vdiRef]: vdi },
+            {}
+          )
```

Comment:
Use [`lodash.pickBy`](https://devdocs.io/lodash~4/index#pickBy) to simplify the test.